src/app/infra/client-storage/local-storage.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { TestBed } from '@angular/core/testing';
+import { LocalStorageService } from './local-storage.service';
+
+describe('LocalStorageService', () => {
+  let localStorageService: LocalStorageService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [LocalStorageService],
+    });
+
+    localStorageService = TestBed.inject(LocalStorageService);
+  });
+
+  afterEach(() => {
+    localStorage.clear();
+  });
+
+  describe('Set', () => {
+    it('should add value in local storage', () => {
+      localStorageService.set('folklore', 'evermore');
+      const value = JSON.parse(localStorage.getItem('folklore') || '');
+      expect(value).toBe('evermore');
+    });
+    it('should throw error when no key is provided to set', () => {
+      expect(() => {
+        localStorageService.set('', 'evermore');
+      }).toThrowError('Error saving to localStorage: key not provided');
+    });
+    it.todo(
+      'should throw error when local storage cannot set the provided value'
+    );
+  });
+  describe('Get', () => {
+    it('should get a single value in local storage', () => {
+      localStorage.setItem('folklore', JSON.stringify('evermore'));
```

Comment:
```suggestion
      localStorage.setItem('folklore', 'evermore');
```

Reply:
https://github.com/iurynogueira/angular-clean/pull/21/files#diff-ddee5c28ad759e54867bb1967bba78c8fa2bcafe3cdb7bdb6b790e384983cd5fR55 you are using here that way

Reply:
Because I'm only testing the remove function there.